### DISCUSSION TITLE
Warning while compiling Swagger-React Autolinker/src/version.ts Doesn't exist

### DISCUSSION THREAD

**Discussion by mattxerini** [swagger-api/swagger-ui#8592] [Q&A]
Hello,
as per title when I compile my project with sweagger-ui-react, I have over 30 warnings in my console about Autolinker

Failed to parse source map from '/node_modules/autolinker/src/version.ts' file: Error: ENOENT: no such file or directory, open '/node_modules/autolinker/src/version.ts'

I am using the latest swagger packages 

  "@types/swagger-ui": "^3.52.0",
  "@types/swagger-ui-dist": "^3.30.1",
  "@types/swagger-ui-react": "^4.18.0",
  "swagger-ui": "^4.10.3",
  "swagger-ui-dist": "^4.18.2",
  "swagger-ui-react": "^4.18.2",

can someone shed some light on those warnings?

**Comment by guilhermemiua** [comment]
Also facing the same problem...

**Comment by mattxerini** [comment]
Anyone from Swagger?

**Comment by yahya-banouk** [comment]
Me too, I'm facing same problem

**Comment by IsaakNazar** [comment]
Same issue

**Comment by guilhermemiua** [comment]
Opened an issue about this: https://github.com/swagger-api/swagger-ui/issues/8967
